manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/plugins/NetworkPoliciesHandler.java

```diff
@@ -70,9 +71,8 @@
             });
 
             // Only Port Number required to generate traffic rules
-            networkTrafficRules.stream().forEach(rule ->
-                    rule.getPorts().stream().forEach(port ->
-                            rule.getPorts().set(rule.getPorts().indexOf(port), port.split("/")[0])));
+            networkTrafficRules.stream().forEach(rule -> rule
+                    .setPorts(rule.getPorts().stream().map(port -> port.split("/")[0]).collect(Collectors.toList())));
```

Comment:
Use PORTS_PROTOCOL_SEPARATOR here as well as wherever we are using "/" for ports